### PR TITLE
Story/geco 1: Nepomuk needs to allow app registration to use its API

### PR DESCRIPTION
**Pull request by snilapwa**

### CODE REVIEW

File: nepomuk/src/main/resources/providers.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+github=GitHub
+mitreidconnect=MITREid Connect Server
```
Comment (jdamerow) [inline nepomuk/src/main/resources/providers.properties]: These are unnecessary

File: nepomuk/src/main/java/edu/asu/diging/gilesecosystem/nepomuk/core/tokens/impl/TokenService.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    /**
+     * 4 hours
+     */
+    private int timeTillExpiration = 14400000;
+    
+    @Autowired
+    private IPropertiesManager propertiesManager;
+
+    @Override
+    public String generateApiToken(User user) {
```
Comment (jdamerow) [inline nepomuk/src/main/java/edu/asu/diging/gilesecosystem/nepomuk/core/tokens/impl/TokenService.java]: I don't think you need this method and the user class.

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: nepomuk/src/main/resources/providers.properties (hunk 1 above)
Comment (snilapwa) [reply nepomuk/src/main/resources/providers.properties]: Removed

File: nepomuk/src/main/java/edu/asu/diging/gilesecosystem/nepomuk/core/tokens/impl/TokenService.java (hunk 2 above)
Comment (snilapwa) [reply nepomuk/src/main/java/edu/asu/diging/gilesecosystem/nepomuk/core/tokens/impl/TokenService.java]: Removed extra functions and User class

**Comment by jdamerow** [comment]
please resolve conflicts

**Pull request closed without merging**